servers/src/main/java/tachyon/worker/block/BlockWorkerUtils.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.worker.block;
+
+import java.net.InetSocketAddress;
+
+import tachyon.Constants;
+import tachyon.conf.TachyonConf;
+import tachyon.util.NetworkUtils;
+
+public final class BlockWorkerUtils {
+
+  /**
+   * Gets the Tachyon master address from the configuration
+   *
+   * @return the InetSocketAddress of the master
+   */
```

Comment:
Please describe the arguments of the method in the java doc (same comment for method below).